contrib/platform/src/com/sun/jna/platform/win32/COM/util/Convert.java

```diff
@@ -21,11 +23,33 @@
 import com.sun.jna.platform.win32.WTypes;
 import com.sun.jna.platform.win32.WinDef;
 import com.sun.jna.platform.win32.Variant.VARIANT;
+import com.sun.jna.platform.win32.WTypes.BSTR;
 
-public class Convert {
-	
+/**
+ * This class is considered internal to the package.
+ */
+class Convert {

```

Comment:
At a minimum, this should probably be named `VARIANTConverter` (a noun) rather than `Convert` (a verb, but also an inappropriate noun).